testing/apt-dater-host/APKBUILD

```diff
@@ -3,23 +3,32 @@
 pkgname=apt-dater-host
 pkgver=1.0.0
```

Comment:
Aha, c3d10e0 is only one commit after 1.0.0 and it adds apk support. Then it’s better to just add it as a patch instead of fetching unreleased source.